scripts/process/process.classified_precip.R

```diff
@@ -0,0 +1,54 @@
+
+precip_to_class <- function(precip, p.times, times.out, breaks, compress = TRUE){
```

Comment:
use `compress=FALSE` if we want the precip bin of each timestep written out completely. FALSE just does the bin diffs.